### PR TITLE
Enhancement / View Fix: Display view range differently on the map from the jump range

### PR DESCRIPTION
**Pull request by RisingLeaf**
**Feature:** This PR implements the feature request detailed and discussed in issue #6676

## Feature Details
The view range is now a hashed line and a little bit thicker than the jump range

## UI Screenshots

Screen Shot 2022-10-11 at 6 59 58 PM

## Testing Done
See screenshot

## Performance Impact
Minimum more of shader calculation time. Not noticeable for normal humans

### CODE REVIEW

**Comment by RisingLeaf** [comment]
for the hacktober fest!!!

**Comment by quyykk** [comment]
It's harder to notice the difference in brighter parts of the map:

image

Maybe dashes and/or a different color would work (either for the jump range or the view range.

**Comment by RisingLeaf** [comment]
Screen Shot 2022-10-15 at 10 29 34 AM
@quyykk This better?

**Comment by RisingLeaf** [comment]
The inner ring is jump range in this case.

File: source/MapPanel.cpp [hunk 1]
```diff
@@ -218,13 +218,14 @@ void MapPanel::Draw()
 
 	// Draw the "visible range" circle around your current location.
 	Color dimColor(.1f, 0.f);
-	RingShader::Draw(Zoom() * (playerSystem.Position() + center),
-		(System::DEFAULT_NEIGHBOR_DISTANCE + .5) * Zoom(), (System::DEFAULT_NEIGHBOR_DISTANCE - .5) * Zoom(), dimColor);
+	RingShader::Draw(Zoom() * (playerSystem.Position() + center), System::DEFAULT_NEIGHBOR_DISTANCE * Zoom(), 2.0f, 1.0f,
+		dimColor, 25.0, 10.0);
```
Comment (quyykk) [inline source/MapPanel.cpp:222]:
```suggestion
		dimColor);
```

The dashing is so low you might as well drop it completely.

**Review by quyykk — APPROVED** [review]

File: source/MapPanel.cpp [hunk 2]
```diff
@@ -218,13 +218,14 @@ void MapPanel::Draw()
 
 	// Draw the "visible range" circle around your current location.
 	Color dimColor(.1f, 0.f);
```
Comment (Amazinite) [inline source/MapPanel.cpp:220]: We should move to defining these colors in the interface file while we're here.

**Review by Amazinite — CHANGES_REQUESTED** [review]

File: source/MapPanel.cpp (hunk 2 above)
Comment (RisingLeaf) [reply source/MapPanel.cpp:220]: will get to it

Comment (RisingLeaf) [reply source/MapPanel.cpp]: done

File: source/MapPanel.cpp [hunk 3]
```diff
@@ -217,14 +217,15 @@ void MapPanel::Draw()
 		FogShader::Draw(center, Zoom(), player);
 
 	// Draw the "visible range" circle around your current location.
-	Color dimColor(.1f, 0.f);
-	RingShader::Draw(Zoom() * (playerSystem.Position() + center),
-		(System::DEFAULT_NEIGHBOR_DISTANCE + .5) * Zoom(), (System::DEFAULT_NEIGHBOR_DISTANCE - .5) * Zoom(), dimColor);
+	const Color &viewRangeColor = *GameData::Colors().Get("map view range color");
+	RingShader::Draw(Zoom() * (playerSystem.Position() + center), System::DEFAULT_NEIGHBOR_DISTANCE * Zoom(), 2.0f, 1.0f,
+					viewRangeColor);
```
Comment (Amazinite) [inline source/MapPanel.cpp:222]:
```suggestion
	RingShader::Draw(Zoom() * (playerSystem.Position() + center),
		System::DEFAULT_NEIGHBOR_DISTANCE * Zoom(), 2.0f, 1.0f, viewRangeColor);
```

**Review by Amazinite — APPROVED** [review]

**Pull request merged**